pkg/katautils/config.go

```diff
@@ -994,6 +993,19 @@ func SetKernelParams(runtimeConfig *oci.RuntimeConfig) error {
 		}
 	}
 
+	// set the scsi scan mode to none for virtio-scsi
+	if runtimeConfig.HypervisorConfig.BlockDeviceDriver == config.VirtioSCSI {
+		if !strings.Contains(formatted, "scsi_mod.scan") {
```

Comment:
@justin-he after reading the code, I think this conditional is not needed, since `kernel_params` overwrites default kernel parameters.

https://github.com/kata-containers/runtime/blob/af24829c2ae78c3b811c4cff6736ddaba500d37c/cli/config/configuration-qemu.toml.in#L24-L30

Reply:
so, in theory this means you can set `scsi_mod.scan=none` by default and the user can change this behaviour

Reply:
Ok I change remove the condition